calendar-overview.adoc

```diff
@@ -63,15 +63,16 @@ Work Week:: Workweek view displays all the events in a _work week_.
 E.g., you are viewing the events in the week *Sunday, 22^nd^ Dec 2019 - Saturday, 28^th^ Dec 2019*, and your work week starts from Monday and ends on Friday.
 Clicking *Work Week* displays events from *Monday, 23^rd^ Dec 2019 - Friday, 27^th^ Dec 2019*.
 +
-NOTE: For better results, set your workweek in <<settings-otheroptions.adoc#_general_calendar_settings, workweek settings>>
```

Comment:
Work week option is present in Modern UI -> Calendar.
The setting for the work week can be configure in Settings -> Calendar and Reminders -> Start of the week. The Work week is set according to the day I update in *Start of the week*.

Reply:
Okay, I understand the current behaviour of the "Work week" option.  I"ll update the text.